integration_test/tests/common/mod.rs

```diff
@@ -150,7 +150,18 @@ pub fn connect_with_conn_string(env_handle: HEnv, in_connection_string: String)
             str_len_ptr,
             DriverConnectOption::NoPrompt,
         ) {
-            SqlReturn::SUCCESS | SqlReturn::SUCCESS_WITH_INFO => (),
+            // Originally, this would return SUCCESS_WITH_INFO since we pass null_mut() as
```

Comment:
I don't think that there is much value in keep some kind of historical comment on a test.
If you want to track the changes, I'd suggest to tag the comment in the feature change itself (in function.rs) with the JIRA ticket which introduced the change.
This help track changes overtime and now which version it was introduced into.